### PR TITLE
added ctrl+enter to private messages #10413

### LINKED ISSUE #10413: Ctrl+Enter and similar hotkeys don't work to send private messages
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

Users can press Ctrl+Enter to send chat messages in Guilds, Parties, and the Tavern without use of the mouse, but this shortcut doesn't work in private messages. No reason it shouldn't!

### PR DESCRIPTION
**Pull request by Peim8**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #10413 
### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

Added a ctrl#enter-shortcut to private messages

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID:  45b66012-a649-489e-8fd0-b9f8a38c2858

### CODE REVIEW

File: website/client/components/userMenu/inbox.vue [hunk 1]
```diff
@@ -353,6 +353,9 @@ export default {
     },
     toggleOpt () {
       this.$store.dispatch('user:togglePrivateMessagesOpt');
+    },
+	async sendMessageShortcut () {
```
Comment (aszlig) [inline website/client/components/userMenu/inbox.vue]: Weird indentation here.

Comment (paglias) [reply website/client/components/userMenu/inbox.vue]: this + this function isn't really necessary as it just calls another one, better use `@keyup.ctrl.enter='sendPrivateMessage()'`

**Review by paglias — CHANGES_REQUESTED** [review]
@PBeier left one comment, if you fix it I'll merge the PR :) thanks!

**Comment by paglias** [comment]
This is working well @PBeier , thanks! 

Note that this week we're not merging contributor PRs because we're doing some internal changes, it'll be merged next week

**Comment by paglias** [comment]
thanks @PBeier ! Notated towards your second tier

**Pull request merged**